remote-api/src/main/java/org/alfresco/rest/framework/resource/parameters/ListPage.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+
+package org.alfresco.rest.framework.resource.parameters;
+
+import org.alfresco.query.PagingResults;
+import org.alfresco.rest.framework.resource.SerializablePagedCollection;
+import org.alfresco.service.Experimental;
+import org.alfresco.util.Pair;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * List page with paging information.
+ *
+ *
+ * @param <E> - list element type
+ */
+@Experimental
+public interface ListPage<E> extends List<E>, PagingResults<E>, SerializablePagedCollection<E>
+{
+
+    default List<E> getPage()
+    {
+        return this;
+    }
+
+    default Pair<Integer, Integer> getTotalResultCount()
```

Comment:
I don't understand why this returns a pair.

Reply:
This is a contract method from `interface PagingResults<R>`. Can't really do much about it.